manager/integration/tests/test_backing_image.py

```diff
@@ -511,6 +514,54 @@ def test_backing_image_auto_resync(client, volume_name):  # NOQA
     6. Wait for the file recovering automatically.
     7. Validate the volume.
     """
+    for bi_url in (BACKING_IMAGE_QCOW2_URL, BACKING_IMAGE_RAW_URL):
```

Comment:
Use https://docs.pytest.org/en/7.1.x/example/parametrize.html instead, so it will run those in separate tests instead of just one and it will make the test result clear.

Reply:
Hi @innobead ,

I modified code by use 
`@pytest.mark.parametrize("bi_url", [BACKING_IMAGE_QCOW2_URL, BACKING_IMAGE_RAW_URL]) # NOQA` and it will run two separate case as you mentioned, thank you